packages/devextreme/js/ui/overlay.d.ts

```diff
@@ -60,13 +52,6 @@ export interface dxOverlayOptions<TComponent> extends WidgetOptions<TComponent>
      * @public
      */
     deferRendering?: boolean;
-    /**
-     * @docid
-     * @deprecated
-     * @default {}
-     * @hidden
-     */
-    elementAttr?: any;
```

Comment:
Please retrive it back - it's needed for MVC wrappers to implement `id` property.
Now I see that `@hidden` doccomment is already here => it's already removed from docs.

The single questions is "do we need @deprecated doccoment here" - i'd prefer to remove it to exclude this from search when we're looking for deprecated API members.

@mpreyskurantov is it okay if we remove it?

Reply:
UPD: I fast synced with Mikhail - let's leave "deprecated" for now